src/screens/TopicMemberScreen/index.js

```diff
@@ -122,7 +137,12 @@ const TopicMemberScreen = () => {
   const showAnimationHeader = () => {
     interactionManagerRef.current = InteractionManager.runAfterInteractions(() => {
       Animated.timing(animatedHeight, {
-        toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT + dimen.size.TOPIC_FEED_HEADER_HEIGHT,
+        toValue:
+          (coverPath
+            ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+            : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
+          dimen.size.TOPIC_FEED_HEADER_HEIGHT +
+          topPosition,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Use the helper function `calculateAnimatedHeight` here as well.
```diff
-  toValue:
-    (coverPath
-      ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-      : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
-    dimen.size.TOPIC_FEED_HEADER_HEIGHT +
-    topPosition,
+  toValue: calculateAnimatedHeight(),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->